Algorithms/ShortestPath.py

```diff
@@ -215,7 +218,7 @@ def reconstruct_path(closed_list: list, current: list) -> tuple:
                         open_list.append([current[1], i, neighbour_g, x, y])
 
         # if the open list is empty and path has not been determined: no path possible, return false
-        return False
+        return ([], )
```

Comment:
Lepsze by było np, ([], -1) czy coś, albo np (None, -1). Do przemyślenia, które z tych lepsze bd. None da błędy, ale wsm jak jest pusta trasa to chyba taki błąd może być pożądany. -1 powinno być dlatego, że to jest czas, więc naturalnie nie może być ujemny